src/erf.jl

```diff
@@ -70,80 +70,83 @@ const sb5  =  2.55305040643316442583e+03 # 0x40A3F219, 0xCEDF3BE6
 const sb6  =  4.74528541206955367215e+02 # 0x407DA874, 0xE79FE763
 const sb7  = -2.24409524465858183362e+01 # 0xC03670E2, 0x42712D62
 
-function erfc1(x::Float64)
+function erfc1{T<:Union{Float32,Float64}}(x::T)
```

Comment:
probably best to just use  a FloatTypes = Union{Float32,Float64} alias  everywhere